packages/core-components/src/lib/atoms/inputs/text/TextInput.svelte

```diff
@@ -0,0 +1,48 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import { INPUTS_CONTEXT_KEY } from '@evidence-dev/component-utilities/globalContexts';
+	import { getContext } from 'svelte';
+	import debounce from 'lodash.debounce';
+	const inputs = getContext(INPUTS_CONTEXT_KEY);
+
+	/////
+	// Component Things
+	/////
+
+	/** @type {string} */
+	export let title;
+
+	/** @type {string} */
+	export let name;
+
+	/**
+	 * @param {string} v
+	 */
+	const updateValue = debounce((v) => {
+		$inputs[name] = {
+			toString() {
+				return v;
+			},
+			sql: `'${v}'`,
+			search: (col) => `damerau_levenshtein(${col}, '${v}')`
+		};
+	}, 100);
```

Comment:
```suggestion
	const updateValue = debounce((text) => {
		$inputs[name] = {
			toString() {
				return text;
			},
			sql: `'${text}'`,
			search: (col) => `damerau_levenshtein(${col}, '${text}')`
		};
	}, 100);
```

Reply:
Should we consider doing something like `text.replaceAll("'", "''")` in `sql` and `search` so people don't accidentally sql inject themselves?

Reply:
> Should we consider doing something like text.replaceAll("'", "''") in sql and search so people don't accidentally sql inject themselves?

:+1:

Reply:
> Also - is there a point to debouncing here? The query-level debouncing should be fine.

This might roll back around to storybook; there likely isn't here - I probably didn't implement debounce on the storybook query stores